constants.txt

```diff
@@ -33,4 +33,11 @@ frobulator_types_e FrobulatorType FROBULATOR_OBTUSE_NAME clearer_name
 # FrobulatorType.obtuse_name.
 
 manual:
-ALSO_GOES_IN_THE_OUTPUT 0xdeadbeef
+# Alias for manual_suffix. These go after the values read from files,
```

Comment:
"Manual suffixes go after the values read from files, and therefore may refer to them"? Is that any clearer? s/them/said values/ ?